src/classes/CanvasRenderingContext2D.js

```diff
@@ -1446,6 +1447,21 @@ export default class CanvasRenderingContext2D {
     this._path.push(event);
   }
 
+  reset() {
+    if (arguments.length > 0) {
+      throw new TypeError(
```

Comment:
Is the information thrown standardized?

Reply:
I'm not sure what you mean by this. I tried to make it match the information in the `rect` function.